microsoft-edge/webview2/reference/javascript/hostobjectasyncproxybase.yml

```
@@ -0,0 +1,136 @@
+### YamlMime:TSType
+name: HostObjectAsyncProxyBase
+uid: 'WebView2Script!HostObjectAsyncProxyBase:class'
+package: WebView2Script!
+fullName: HostObjectAsyncProxyBase
+summary: >-
+  An asynchronous host object proxy. Host objects added via `CoreWebView2.AddHostObjectToScript` are exposed as host
+  object proxies using `window.chrome.webview.hostObjects.{name}`<!-- -->. Host object proxies are promises and resolve
+  to an object representing the host object. The promise is rejected if the app has not added an object with the name.
+  When JavaScript code access a property or method of the object, a promise is returned, which resolves to the value
+  returned from the host for the property or method, or rejected in case of error, for example, no property or method on
+  the object or parameters are not valid.
+
+
+  Host object proxies are JavaScript Proxy objects that intercept all property get, property set, and method
+  invocations. Properties or methods that are a part of the Function or Object prototype are run locally. Additionally

```

```suggestion
  invocations. Properties or methods that are a part of the `Function` or `Object` prototype are run locally. Additionally
```
if a lang keyword, take opportunity to offset from regular English words.